scalyr_agent/log_processing.py

```diff
@@ -1646,6 +1651,7 @@ def perform_processing(self, add_events_request, current_time=None):
                     if not add_events_request.add_event(event, timestamp=line_object.timestamp, sequence_id=sequence_id, sequence_number=sequence_number):
                         #time_spent_serializing -= fast_get_time()
 
+                        log.info('Resetting file position to %d (not add_events_request)' % position.mark_offset)
```

Comment:
Don't need to include this log line as it was only for testing and we don't this filling real logs (this path will happen almost every time for high volume logs), plus it was for the negative bytes issue anyway.